app/src/main/java/com/example/admin/showdata/Content_model.java

```diff
@@ -0,0 +1,67 @@
+package com.example.admin.showdata;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by admin on 15-02-2018.
+ */
+
+public class Content_model {
```

Comment:
Always follow Android Naming convention: https://github.com/ribot/android-guidelines/blob/master/project_and_code_guidelines.md

Reply:
got it